deploy/cstor-operator.yaml

```diff
@@ -115,6 +115,23 @@ spec:
         - name:  OPENEBS_IO_VOLUME_MONITOR_IMAGE
           value: "openebs/m-exporter:ci"
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cvc-operator-service
+  namespace: openebs
+  labels:
+    openebs.io/component-name: cvc-operator-svc
```

Comment:
Is this the correct component name?

Reply:
Yes, `openebs.io/component-name` will convey what kind of openebs component it is.
Ref: https://github.com/openebs/charts/blob/b8474d2c7f132303ad8852cab0e629038c18d40b/openebs-operator-1.10.0.yaml#L240

Reply:
This label needs to be present on cvc-operator I suppose. 
Is it present?
cc @prateekpandey14

Reply:
Ok. I think I got it.

Reply:
Ignore the comment